### ISSUE TITLE
STIR #325 breaks SIRF FBP2D

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#354]
https://github.com/UCL/STIR/pull/325 ([specific commit](https://github.com/UCL/STIR/pull/325/commits/911f01c3965c7e1dfb17b715d6adf92f13f4c193)) moved the zoom etc variables to a separate class ParseDiscretisedDensityParameters which is now a member of AnalyticReconstruction etc. There were good reasons to do it :-;

error
```
In file included from /home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/SIRF/src/xSTIR/cSTIR/cstir_p.cpp:27:0:
/home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/SIRF/src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h: In member function â€˜void sirf::xSTIR_FBP2DReconstruction::set_zoom(double)â€™:
/home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/SIRF/src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h:612:4: error: â€˜zoomâ€™ was not declared in this scope
    zoom = z;
    ^~~~
/home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/SIRF/src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h: In member function â€˜void sirf::xSTIR_FBP2DReconstruction::set_output_image_size_xy(int)â€™:
/home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/SIRF/src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h:616:4: error: â€˜output_image_size_xyâ€™ was not declared in this scope
    output_image_size_xy = xy;
    ^~~~~~~~~~~~~~~~~~~~
```

**Comment by KrisThielemans** [comment]
This means that 
- SIRF-SuperBuild `DEVEL_BUILD` (and of course 2.0) will need to upgrade the STIR commit in version_config.cmake (but that had to be done anyway)
- we can no longer rely on `xSTIR_FBP2DReconstruction` to have access to the relevant members, as currently assumed in  https://github.com/CCPPETMR/SIRF/blob/c64e3d0658096097c6d205be85d80e3f528d046c/src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h#L612. It seems to me we need [stir::AnalyticReconstruction](https://github.com/UCL/STIR/blob/master/src/include/stir/recon_buildblock/AnalyticReconstruction.h) to provide the relevant `set` (and `get`?) functions, which would have made more sense anyway. I'll create a STIR issue for that.

**Comment by rijobro** [comment]
Is it alright if I modify `versions_config.cmake` to point to the STIR commit before this problem for `DEVEL_BUILD=ON`? Once this issue is closed, we can revert back to using master again.

**Comment by KrisThielemans** [comment]
fixed with #367
